### PR TITLE
Landing page finished

### PR DESCRIPTION
**Pull request by Hai5edfm**
Were added some scss global things, the landing page reponsive, and a provisional (dev) dependency to load files (like png/jpg) (in the future with the DB we will change it)

image
image
image

### CODE REVIEW

File: dist/0aa1f5c9ad027e95e2ba.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
...
...SON.stringify(a))))," */")),t.styleTagTransform(r,e,t.options)}(t,e,n)},remove:function(){!function(e){if(null===e.parentNode)return!1;e.parentNode.removeChild(e)}(t)}}}},589:function(e){e.exports=function(e,t){if(t.styleSheet)t.styleSheet.cssText=e;else{for(;t.firstChild;)t.removeChild(t.firstChild);t.appendChild(document.createTextNode(e))}}}},t={};function n(r){var l=t[r];if(void 0!==l)return l.exports;var a=t[r]={id:r,exports:{}};return e[r](a,a.exports,n),a.exports}n.n=function(e){var t=e&&e.__esModule?function(){return e.default}:function(){return e};return n.d(t,{a:t}),t},n.d=function(e,t){for(var r in t)n.o(t,r)&&!n.o(e,r)&&Object.defineProperty(e,r,{enumerable:!0,get:t[r]})},n.o=function(e,t){return Object.prototype.hasOwnProperty.call(e,t)},function(){var e=n(294),t=n(935),r=n(379),l=n.n(r),a=n(795),o=n.n(a),u=n(569),i=n.n(u),c=n(565),s=n.n(c),f=n(216),d=n.n(f),p=n(589),h=n.n(p),m=n(429),v={};v.styleTagTransform=h(),v.setAttributes=s(),v.insert=i().bind(null,"head"),v.domAPI=o(),v.insertStyleElement=d(),l()(m.Z,v),m.Z&&m.Z.locals&&m.Z.locals,t.render(e.createElement((function(){return e.createElement("h1",null,"Hello world!")}),null),document.getElementById("App"))}()}();
```
Comment (AreYouIvan) [inline dist/0aa1f5c9ad027e95e2ba.js:2]: Are you starting npm as production? or what is this file in the dist folder?

File: src/components/GetHttpURL/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```
Comment (AreYouIvan) [inline src/components/GetHttpURL/index.js:1]: Excuse me, sir, what is the functionality of this component?

File: src/pages/LandingPage/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { LoginButton } from '../../components/LoginButton';
+import { CTA_RegisterButton } from '../../components/RegisterButton';
```
Comment (AreYouIvan) [inline src/pages/LandingPage/index.js:3]: What CTA does mean? 😋

File: src/routes/css/Mixins/buttons.scss [hunk 4]
```diff
@@ -0,0 +1,9 @@
+@import '../../App.scss';
+@mixin BasicButton--LightTheme {
```
Comment (AreYouIvan) [inline src/routes/css/Mixins/buttons.scss:2]: Cool! we already have the button style variable 😎

**Review by AreYouIvan — COMMENTED** [review]
There are a lot of files with woff extension (fonts?) in dist folder, why they're there?

File: src/pages/LandingPage/index.js (hunk 3 above)
Comment (Hai5edfm) [reply src/pages/LandingPage/index.js:3]: Call To Action xd the button of the landing page which is there to captivate the attention of the user and lead them to make an action xd, in this case, register :v

File: src/components/GetHttpURL/index.js (hunk 2 above)
Comment (Hai5edfm) [reply src/components/GetHttpURL/index.js:1]: I was / I am yet. xd trying to get the document.location.href to compare it with that href with the link that the buttons has :v and also I know if that could works with the back button, but I'm still investigating the snapshot testing and those situations

File: dist/0aa1f5c9ad027e95e2ba.js (hunk 1 above)
Comment (Hai5edfm) [reply dist/0aa1f5c9ad027e95e2ba.js:2]: No se si se deba a que importe la fuente 'Poppins' desde el HTML

**Review by AreYouIvan — APPROVED** [review]

**Pull request closed without merging**